kwalitee/cli.py

```diff
@@ -116,3 +134,34 @@ def generate_tool_json(ctx, output=None):
         Path(output).write_text(json_str)
     else:
         print(json_str)
+
+
+@cli2.command('validate', help='''
+
+    Validate created JSON files.
+
+''')
+@click.option('-f', '--file',
+    type=click.Choice(['repos,json', 'ocrd_all_releases.json'], case_sensitive=True),
```

Comment:
```suggestion
    type=click.Choice(['repos.json', 'ocrd_all_releases.json']),
```

`case_sensitive=True` is the default.